tests/test_bdist_wheel.py

```diff
@@ -421,3 +422,25 @@ def test_platform_linux32(reported, expected, monkeypatch):
     cmd.root_is_pure = False
     _, _, actual = cmd.get_tag()
     assert actual == expected
+
+
+def test_no_ctypes(monkeypatch) -> None:
+    builtins_import = builtins.__import__
+
+    def _fake_import(name: str, *args, **kwargs):
+        if name == "ctypes":
+            raise ModuleNotFoundError("No module named %s" % name)
+
+        return builtins_import(name, *args, **kwargs)
```

Comment:
If you want one more tiny simplification, you could use `importlib.__import__` here, removing the need to save the original `__import__`. This is a pretty standard idiom though so I think it’s also fine.

Reply:
Thanks you!